### ISSUE TITLE
Don`t paint chart

### ISSUE THREAD

**Issue by NickSavin** [beto-rodriguez/Live-Charts#262]
Hello, I have got a new issue.
Try to describe =)

When TabItem with a chart in Focus, chart is not painted.

![image](https://cloud.githubusercontent.com/assets/20172765/16868381/febee280-4a7f-11e6-9bb0-86c89030580a.png)

`
                            <TabItem Header="Chart">

                                <lvc:CartesianChart LegendLocation="Top"  Margin="5" Padding="5" Series="{Binding OutputVM.BudgetToIntenet.ChartSeries}" >

                                    <lvc:CartesianChart.ChartLegend>
                                        <lvc:DefaultLegend Margin="0,0,0,10"/>
                                    </lvc:CartesianChart.ChartLegend>                                  
                                
                                </lvc:CartesianChart>

                            </TabItem>
`
a part of the calculation method:
`

            ChartSeries.Clear();

            var AdditionalTVReachSER = new LiveCharts.Wpf.LineSeries() { Values = new ChartValues<ObservablePoint>(), Title = "Additional TV Reach Point Cost" };
            var CPIIncrementEffectiveReachSER = new LiveCharts.Wpf.LineSeries() { Values = new ChartValues<ObservablePoint>(), Title= "Cost per Internet Increment Effective Reach Point" };
            var EffectiveReachSER = new LiveCharts.Wpf.LineSeries() { Values = new ChartValues<ObservablePoint>(), Title = "Effective Reach Increment by Internet" };

            foreach (var item in BudgetToIntenetModel.Data)
            {
                AdditionalTVReachSER.Values.Add(new ObservablePoint((double)Math.Round(item.BudgetInternet), (double)Math.Round(item.AdditionalTVReachPointCost)));
                CPIIncrementEffectiveReachSER.Values.Add(new ObservablePoint((double)Math.Round(item.BudgetInternet), (double)Math.Round(item.CostPerInternetIncrementEffectiveReachPoint)));
                EffectiveReachSER.Values.Add(new ObservablePoint((double)item.BudgetInternet, item.EffectiveReachIncrementByInternet));
            }

            ChartSeries.Add(AdditionalTVReachSER);
            ChartSeries.Add(CPIIncrementEffectiveReachSER);
            ChartSeries.Add(EffectiveReachSER);

`
collection is:
public LiveCharts.SeriesCollection ChartSeries { get; set; } = new LiveCharts.SeriesCollection();

Then:

When I execute method second time at this TabItem, result is NRE
![image](https://cloud.githubusercontent.com/assets/20172765/16868668/9fac74d6-4a81-11e6-86ea-ecdf03c01992.png)

But When the other TabItem in Focus, as on screenshot, It is all right. chart is painted
![image](https://cloud.githubusercontent.com/assets/20172765/16868341/c7c2ce40-4a7f-11e6-8936-61082dda4710.png)

![image](https://cloud.githubusercontent.com/assets/20172765/16868487/88b3bc5e-4a80-11e6-99f4-a748c8e7938b.png)

This problem appears when 
Series="{Binding OutputVM.BudgetToIntenet.ChartSeries}" 

If I define every series in XAML and bind to its Values property, its all right to.

But in my case I cant do thus, because my  series type depend on calculation method

**Comment by NickSavin** [comment]
The Axes will be binded in the same manner (by binding the axes collection), when a new ver. of library will be committed. 
depend on  #258

is there will  be #262 issue?

Thanks!

**Comment by beto-rodriguez** [comment]
I just published 0.7.7 I made some changes there, to fix #257 could you please verify if the issue is fixed in this case?

Thanks!

**Comment by NickSavin** [comment]
Update to 078, but a problem is still here

**Comment by NickSavin** [comment]
Can this issue appear because of trying to access SeriesCollection from another thread?

My Calculation method is going at the new thread.
When I need to access SeriesCollection, I use Dispatcher

`
`
                Application.Current.Dispatcher.Invoke(() =>
                {
                    ChartSeries.Clear();

                    foreach (var item in data)
                    {
                        var lineSeries = ChartSeries.SingleOrDefault(s => ((LineSeries)s).Title == item.PublisherName);

                        if (lineSeries==null)
                        {
                            lineSeries = new LineSeries() { Title = item.PublisherName, Values = new LiveCharts.ChartValues<LiveCharts.Defaults.ObservablePoint>() };
                            ChartSeries.Add(lineSeries);
                        }

                        lineSeries.Values.Add(new LiveCharts.Defaults.ObservablePoint((double)(Math.Round(item.Budget)), item.Reach));
                    }                   

                });

`
`

in this case issue appears

**Comment by NickSavin** [comment]
If I will not use Dispatcher, I will got an exeption

"Вызывающим потоком должен быть STA, поскольку этого требуют большинство компонентов UI."  "the caller thread must be STA , because it needs the most of UI components"

**Comment by beto-rodriguez** [comment]
Please try 0.7.9 it should work correctly now.

Thank you very much for this detailed report!
